packages/cli/src/commands/add/isListablePackage.ts

```diff
@@ -8,7 +8,7 @@ export function isListablePackage(config: Config, packageJson: PackageJSON) {
     return false;
   }
 
-  if (!config.privatePackages && packageJson.private) {
+  if (!config.privatePackages?.version && packageJson.private) {
```

Comment:
Could you add a test case that would cover the fix?

Isn't `config.privatePackages` always an object? If yes, shouldn't it be this?
```suggestion
  if (!config.privatePackages.version && packageJson.private) {
```

Reply:
i have added test case, is that ok?